recipes/super-collections/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "super-collections" %}
+{% set pypi_name = "super_collections" %}
```

Comment:
PyPI uses both names, in conda, there is only one. This extra jinja is confusing and what we recommend is:

Use the underscore version, b/c that matches the new URL in PyPI. Or add a multiple output to create both recipes. Here is an example: https://github.com/conda-forge/ioos_metrics-feedstock/blob/main/recipe/meta.yaml